src/legacy/core_plugins/kibana/public/home/components/add_data.js

```diff
@@ -250,8 +264,7 @@ const AddDataUi = ({ apmUiEnabled, isNewKibanaInstance, intl, mlEnabled }) => {
               </EuiLink>
             </EuiText>
           </EuiFlexItem>
-          : null
-        }
+        ) : null}
```

Comment:
Optional, you can do `{mlEnabled && (` without having to result to a ternary.

Reply:
But if this is just the linter making changes I can see leaving this code alone for now.

Reply:
Yeah, more linter craziness :)